CADETPythonSimulator/system_solver.py

```diff
@@ -34,21 +34,25 @@ def _setup_unit_operations(self, unit_operations):
         for i_unit, unit in enumerate(self.unit_operations):
             for port in range(unit.n_outlet_ports):
                 origin_unit_ports[i_unit][port] = origin_counter
-                origin_index_unit_operations[origin_counter] = {'unit': i_unit, 'port': port}
+                origin_index_unit_operations[origin_counter] = {
+                    'unit': i_unit, 'port': port
+                    }
```

Comment:
fix indentation

Reply:
system_solver.py will be removed regardless.